deeplake/integrations/mmdet/mmdet_.py

```diff
@@ -1307,6 +1308,15 @@ def _train_detector(
     auto_scale_lr(cfg, distributed, logger)
     optimizer = build_optimizer(model, cfg.optimizer)
 
+    cfg.custom_imports = dict(
+        imports=["deeplake.integrations.mmdet.mmdet_runners"],
+        allow_failed_imports=False,
+    )
```

Comment:
MMDet mostly uses mmcv under the hood, without this import you will not be able to use custom defined classes.